src/utils/init-project/sushi-config.yaml

```diff
@@ -1,22 +1,195 @@
-# ╭──────────────────────────────────────ImplementationGuide───────────────────────────────────────╮
-# │  The properties below are used to create the ImplementationGuide resource. For a list of       │
-# │  supported properties, see: https://fshschool.org/docs/sushi/configuration/                    │
+# ╭─────────────────────────Commonly Used ImplementationGuide Properties───────────────────────────╮
+# │  The properties below are used to create the ImplementationGuide resource. The most commonly   │
+# │  used properties are included. For a list of all supported properties and their functions,     │
+# │  see: https://fshschool.org/docs/sushi/configuration/.                                         │
 # ╰────────────────────────────────────────────────────────────────────────────────────────────────╯
 id: fhir.example
 canonical: http://example.org
 name: ExampleIG
-status: draft
+# title: Example Title
+# description: Example Implementation Guide for getting started with SUSHI
+status: draft # draft | active | retired | unknown
 version: 0.1.0
-fhirVersion: 4.0.1
+fhirVersion: 4.0.1 # https://www.hl7.org/fhir/valueset-FHIR-version.html
 copyrightYear: 2020+
-releaseLabel: ci-build
+releaseLabel: ci-build # ci-build | draft | qa-preview | ballot | trial-use | release | update | normative+trial-use
+# license: CC0-1.0 # https://www.hl7.org/fhir/valueset-spdx-license.html
+# jurisdiction: urn:iso:std:iso:3166#US "United States of America" # https://www.hl7.org/fhir/valueset-jurisdiction.html
 publisher:
   name: Example Publisher
   url: http://example.org/example-publisher
 
+# The dependencies property corresponds to IG.dependsOn. The key is the
+# package id and the value is the version (or dev/current). For advanced
+# use cases, the value can be an object with keys for `uri` and `version`.
```

Comment:
This is a minor discrepency between the exhaustive example on FSH School and the example config in the SUSHI repo, but the value can be an object with keys for `uri`, `version`, and `id`. The SUSHI example config lists all three in the comment above. Do you think we should include `id` here as well? It's pretty clearly commented out below, so it probably doesn't matter much.

Reply:
Un-discrepancied in https://github.com/FHIR/sushi/pull/944/commits/648a5d0cf98d797958a3096f1e93d52720b262aa